src/platform/linux/x11/window.rs

```diff
@@ -380,6 +380,34 @@ impl UnownedWindow {
                 window.set_always_on_top_inner(window_attrs.always_on_top).queue();
             }
 
+            if window_attrs.center_window && !window_attrs.fullscreen.is_some() 
+            {
+                let window_size = window.get_outer_size();
+
+                let monitor_size;
+                let monitor_position;
+
+                if window_attrs.start_monitor.is_none() {
+                    let monitor = window.get_primary_monitor();
+                    monitor_size = monitor.get_dimensions();
+                    monitor_position = monitor.get_position();
+                } else {
+                    let monitor = window_attrs.start_monitor.unwrap().inner;
+                    monitor_size = monitor.get_dimensions();
+                    monitor_position = monitor.get_position();
+                }
+
+
+                let mut monitor_window_position: LogicalPosition = (0.0, 0.0).into();
+                monitor_window_position.x =
+                    monitor_position.x + (monitor_size.width * 0.5) - (&window_size.unwrap().width * 0.5);
+                monitor_window_position.y =
+                    monitor_position.y + (monitor_size.height * 0.5) - (&window_size.unwrap().height * 0.5);
```

Comment:
What if `monitor_position.{x,y}` != 0? Won't this not centre the window?

Reply:
I don't see any problem here, it centers on both of my monitors. have tried entering custom values to test, but that works as expected.

Reply:
Ah, I thought `monitor_position` was the window's position on the monitor. Looking at the code again, I realize I was mistaken.